### PR TITLE
Adding svmbir

### PR DESCRIPTION
**Pull request by bwohlberg**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sjkisner** [comment]
@sjkisner  I'm willing to be listed in the maintainer section.

**Comment by gbuzzard** [comment]
I'm willing to be listed in the maintainer section.

**Comment by cabouman** [comment]
I'm willing to be listed in the maintainer section.

File: recipes/svmbir/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy =1.21
```
Comment (xhochy) [inline recipes/svmbir/meta.yaml:21]: This should only be in host
```suggestion
```

File: recipes/svmbir/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  host:
+    - conda-build
```
Comment (xhochy) [inline recipes/svmbir/meta.yaml:23]:
```suggestion
```

File: recipes/svmbir/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - llvm-openmp  # [osx]
+    - cython
+    - pip
+    - python
+    - numpy =1.21
+  run:
+    - _openmp_mutex  # [linux]
```
Comment (xhochy) [inline recipes/svmbir/meta.yaml:30]:
```suggestion
```

File: recipes/svmbir/meta.yaml (hunk 1 above)
Comment (bwohlberg) [reply recipes/svmbir/meta.yaml:21]: These were all added as part of attempts to resolve build errors, but I'll try them one by one to check if they're really needed.

File: recipes/svmbir/meta.yaml (hunk 3 above)
Comment (bwohlberg) [reply recipes/svmbir/meta.yaml]: Based on the build failure, it appears that this one was necessary.

Comment (xhochy) [reply recipes/svmbir/meta.yaml:28]: No, then you might need libgomp in host

Comment (bwohlberg) [reply recipes/svmbir/meta.yaml]: That seems to work.

**Pull request merged**